front-end/src/ui-components/AccountSelectionForm.tsx

```diff
@@ -31,6 +33,9 @@ const AccountSelectionForm = ({ title, accounts, address, className, onAccountCh
 			defaultAddress={address || accounts[0]?.address}
 			onAccountChange={onAccountChange}
 		/>
+		{balance &&
+			<Balance address={address} />
```

Comment:
if we pull the balance in this component, let's integrate it better then. The styling now is not different than before.

Reply:
Something like this maybe? Was hesitant as nothing else is right-aligned, but feels right to me
<img width="422" alt="Screenshot 2020-04-14 at 14 05 09" src="https://user-images.githubusercontent.com/7072141/79223221-34212d00-7e59-11ea-960d-6e23d9106f97.png">

Reply:
yup, that looks good.